status-im/status-react
# chore: add generate code to new sync ui

fixes https://github.com/status-im/status-mobile/issues/15548

This implements the second page of the new sync screens. 

please not that the authentication process is still using the old bottom sheet and designs as the new authentication process designs are still being created.

<img width="319" alt="Screenshot 2023-04-03 at 21 18 46" src="https://user-images.githubusercontent.com/22799766/230070901-317ef086-b5d7-442a-a04c-e92f6a6e6234.png">

To test:
Navigate to this page,
click the generate sync code button, enter your password and then it should create the sync code.
After 1:59 the sync code should be disappear and the screen will reset
